### ISSUE TITLE
Intelij debugger evaluations with line feeds fail, must manually remove

### ISSUE THREAD

**Issue by bedge** [BashSupport-Pro/bashsupport-pro#130]
It's common to want to see how a command looks with the variables removed, eg:
```
aws ec2 copy-image \
        --region "${AWS_REGION}" \
        --name "${ARTIFACT_NAME}" \
        --source-region "${source_region}" \
        --source-image-id "${ARTIFACT_ID}" \
        --copy-image-tags \
        --description "Replicated from ${source_region}:${ARTIFACT_NAME}" \
        --output text
```

When this is highlighted and one does the Alt-F8 "eval" command, it retains the embedded line feeds and the result is always empty:
![ss-2023-44-24_08 44 48](https://user-images.githubusercontent.com/499317/227575472-2adfe8f6-300a-4bac-9135-82b1ea0d877c.png)

However, if one manually removes every newline and it's associated `\`, you get the expected result:
![ss-2023-45-24_08 45 12](https://user-images.githubusercontent.com/499317/227575792-934f6a5e-7f18-419e-9a71-fea45eb08444.png)

I would ask that this action be an automatic part of the "evaluation" process.

One should not need to edit the string being evaluated just to remove whitespace that is breaking the ability to evaluate.

context:

BashSupportPro: 3.2.1.223
IntelliJ IDEA 2022.3.3 (Ultimate Edition)

thanks

**Comment by jansorg** [comment]
Thanks! 
Evaluation is probably not escaping the line feeds properly. I'll look into this for the next update.

**Comment by bedge** [comment]
Yup, that sounds like it exactly. Thanks

**Comment by bedge** [comment]
One other observation. In the evaluation dialog history, the LF's are removed, so if one add the eval, then opens the eval history to re-eval, it's correct.
Not the ideal workflow, but better than pulling out 20 LF's by hand.
Perhaps that tidbit could lead to using the same escaping method that the history uses, only before it get's pushed onto the stack?

**Comment by jansorg** [comment]
@bedge I've fixed the handling of line continuations (i.e. `\` followed by a line-feed) for the next bug fix update. I couldn't replicate the behavior of the history you mentioned, but everything seems to working as expected with the fix applied.

**Comment by jansorg** [comment]
This should be fixed by 3.2.2, which was released a few weeks ago.
